src/app/form-entry/directives/historical-value-directive.ts

```diff
@@ -44,12 +44,14 @@ export class HistoricalValueDirective {
 
       if (this.question.hasOwnProperty('hasHistoricalValue')) {
 
-        if (this.question.renderingType === 'select' && !_.isUndefined(this.question.originalValue)) {
+        if ((this.question.renderingType === 'select'
+          || this.question.renderingType === 'multi-select')
+          && !_.isUndefined(this.question.originalValue)) {
 
           this.question['historicalDisplay'] = this.historicalFieldHelper.getDisplayTextFromOptions(
             this.question.options,
             'value',
-            this.question.originalValue.value,
+            this.question.originalValue,
```

Comment:
My only comment is to move originalValue to the 'Nodes' since this will not work when it comes to repeating groups. @enyachoke is storing the original value or initial value in the Nodes. You might want to talk to him.  Also default values is reserved for fields which have default values. For e.g. the default value for 'Start of the week question' would be 'Monday'.